Dockerfile

```diff
@@ -68,6 +80,8 @@ COPY  --from=sail-build /install/cheriot_sim /cheriot-tools/bin/
 COPY  --from=ibex-build cheriot_ibex_safe_sim /cheriot-tools/bin/
 COPY  --from=ibex-build cheriot_ibex_safe_sim_trace /cheriot-tools/bin/
 COPY  --from=cheriot-audit /cheriot-audit/build/cheriot-audit /cheriot-tools/bin/
+# Install the mpact simulator
+COPY  --from=mpact-build /mpact-cheriot/bazel-bin/cheriot/mpact_cheriot /cheriot-tools/bin/
```

Comment:
Does the simulator have any other run-time dependencies?

Reply:
No other run-time dependency. Do look at the run-time options though, including -i for interactive (debug).